less/components/expander.less

```diff
@@ -21,6 +21,9 @@ html.no-js {
     font-weight: bold;
     text-transform: uppercase;
     padding: 2rem;
+    margin: 0;
+    font-size: inherit;
```

Comment:
@sabrinang 
FYI, this is the "Expander" we are talking about

![screen shot 2015-05-19 at 4 14 42 pm](https://cloud.githubusercontent.com/assets/2896608/7715969/3453aab0-fe42-11e4-9f88-50aea2a5aa39.png)

You can find them on https://teach.mozilla.org/teach-like-mozilla/web-literacy/ and https://teach.mozilla.org/clubs/toolkit/